template/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -690,6 +683,12 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				LD_RUNPATH_SEARCH_PATHS = "/usr/lib/swift $(inherited)";
```

Comment:
Thanks for looking into that!

So, by using those two lines, we don't have to "force" Swift? Interesting!

Reply:
Yup! 👍 

(And the above dead code stripping, but that’s only because it was override from the default.)